modules/HapticGlove_module/src/GloveWearable.cpp

```diff
@@ -354,7 +354,7 @@ bool GloveWearableImpl::setFingertipForceFeedbackValues(const std::vector<int>&
         wearableActuatorCommand.info.status = wearable::msg::ActuatorStatus::OK;
         wearableActuatorCommand.duration = 0;
 
-        m_iWearActuatorPort.write(false);
+        m_iWearActuatorPort.write(true);
```

Comment:
Can you add a line of documentation why this is needed? The same below

Reply:
Done.